### PR TITLE
feat(Stickers): sticker packs, sticker routes, and guild stickers

### PR DESCRIPTION
**Pull request by advaith1**
https://github.com/discord/discord-api-docs/pull/3127
https://github.com/discord/discord-api-docs/pull/3128

can review now but don't merge until second API docs pr is merged

### CODE REVIEW

**Review by kyranet — APPROVED** [review]

File: payloads/v8/sticker.ts [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+/**
+ * https://discord.com/developers/docs/resources/sticker#sticker-object-sticker-types
+ */
+export const enum StickerType {
+	/**
+	 * An official sticker in a pack, part of Nitro or in a removed purchasable pack
+	 */
+	STANDARD = 1,
```
Comment (vladfrangu) [inline payloads/v8/sticker.ts:74]: We use PascalCase for enum members

File: payloads/v8/sticker.ts [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+	 * A sticker uploaded to a Boosted guild for the guild's members
+	 */
+	GUILD,
+}
+
+/**
+ * https://discord.com/developers/docs/resources/channel#sticker-object-sticker-format-types
+ */
+export const enum StickerFormatType {
+	PNG = 1,
```
Comment (vladfrangu) [inline payloads/v8/sticker.ts:85]: Ditto

File: payloads/v9/sticker.ts [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+}
+
+/**
+ * https://discord.com/developers/docs/resources/sticker#sticker-object-sticker-types
+ */
+export const enum StickerType {
+	/**
+	 * An official sticker in a pack, part of Nitro or in a removed purchasable pack
+	 */
+	STANDARD = 1,
```
Comment (vladfrangu) [inline payloads/v9/sticker.ts:74]: Ditto

File: rest/v8/sticker.ts [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	 */
+	name?: string;
+	/**
+	 * Description of the sticker (up to 100 characters)
+	 */
+	description?: string;
+	/**
+	 * The name of a unicode emoji representing the sticker's expression
+	 */
+	tags: string;
```
Comment (vladfrangu) [inline rest/v8/sticker.ts:46]: Is this required?

**Review by vladfrangu — CHANGES_REQUESTED** [review]

File: rest/v8/sticker.ts (hunk 4 above)
Comment (advaith1) [reply rest/v8/sticker.ts:46]: yeah https://github.com/discord/discord-api-docs/pull/3127#discussion_r652056949

File: payloads/v8/sticker.ts (hunk 2 above)
Comment (advaith1) [reply payloads/v8/sticker.ts:85]: this was already capitalized before, I'll change it

File: rest/v8/sticker.ts (hunk 4 above)
Comment (advaith1) [reply rest/v8/sticker.ts:46]: wait oops this is PATCH... shouldn't be

**Review by vladfrangu — APPROVED** [review]

**Review by kyranet — APPROVED** [review]

File: payloads/v8/channel.ts [hunk 5]
```diff
@@ -346,6 +341,19 @@ export interface APIMessage {
...
+	 * See https://discord.com/developers/docs/resources/sticker#sticker-item-object
+	 */
+	sticker_items?: APIStickerItem[];
+	/**
+	 * The stickers sent with the message
+	 *
+	 * See https://discord.com/developers/docs/resources/sticker#sticker-object
+	 * @deprecated Use `sticker_items` instead
+	 */
+	stickers?: APISticker[];
```
Comment (vladfrangu) [inline payloads/v8/channel.ts:356]: Shouldn't we just remove these?

File: payloads/v9/channel.ts [hunk 6]
```diff
@@ -378,14 +373,27 @@ export interface APIMessage {
...
 	 */
-	thread?: APIChannel;
+	sticker_items?: APIStickerItem[];
+	/**
+	 * The stickers sent with the message
+	 *
+	 * See https://discord.com/developers/docs/resources/sticker#sticker-object
+	 * @deprecated Use `sticker_items` instead
+	 */
+	stickers?: APISticker[];
```
Comment (vladfrangu) [inline payloads/v9/channel.ts:396]: Dito

File: rest/v8/channel.ts [hunk 7]
```diff
@@ -200,6 +200,12 @@ export interface RESTPostAPIChannelMessageJSONBody {
 	 * See https://discord.com/developers/docs/resources/channel#message-object-message-reference-structure
 	 */
 	message_reference?: APIMessageReferenceSend;
+	/**
+	 * IDs of up to 3 stickers in the server to send in the message
+	 *
+	 * See https://discord.com/developers/docs/resources/sticker#sticker-object
+	 */
+	sticker_ids?: Snowflake[];
```
Comment (vladfrangu) [inline rest/v8/channel.ts:208]:
```suggestion
	sticker_ids?: [Snowflake] | [Snowflake, Snowflake] | [Snowflake, Snowflake, Snowflake];
```

File: rest/v9/channel.ts [hunk 8]
```diff
@@ -228,6 +228,12 @@ export interface RESTPostAPIChannelMessageJSONBody {
 	 * See https://discord.com/developers/docs/resources/channel#message-object-message-reference-structure
 	 */
 	message_reference?: APIMessageReferenceSend;
+	/**
+	 * IDs of up to 3 stickers in the server to send in the message
+	 *
+	 * See https://discord.com/developers/docs/resources/sticker#sticker-object
+	 */
+	sticker_ids?: Snowflake[];
```
Comment (vladfrangu) [inline rest/v9/channel.ts:236]:
```suggestion
	sticker_ids?: [Snowflake] | [Snowflake, Snowflake] | [Snowflake, Snowflake, Snowflake];
```

**Review by vladfrangu — CHANGES_REQUESTED** [review]

**Comment by kyranet** [comment]
This needs a rebase

**Review by vladfrangu — APPROVED** [review]

**Pull request merged**